README.md

```diff
@@ -36,16 +36,20 @@ To customize your test suite configuration you can add a few more options to php
 ```xml
 <php>
     <server name="KERNEL_DIR" value="/path/to/dir/with/kernel" />
```

Comment:
Yup. It is used by basic WebTestCase.  We will have it as an optional parameter.